### ISSUE TITLE
Can I convert ALL my SVG files to 512px height

### ISSUE THREAD

**Issue by thybzi** [yoksel/grunt-svg-modify#4]
Just wish to fix issues like that: https://github.com/sapegin/grunt-webfont/issues/222 and https://github.com/sapegin/grunt-webfont/issues/219

Does the module have ability to resize all my SVG files to 512px height?

Заранее спасибо :)

**Comment by localBlackMage** [comment]
@thybzi  I came across this same issue myself when trying to implement this plugin. I'm not impressed with the suggested fix as it relies on a less than reliable tweak to make the plugin generate correct output.

**Comment by thybzi** [comment]
@hprofit But I cannot find any decent grunt-compatible alternative for grunt-webfont :(

**Comment by localBlackMage** [comment]
@thybzi   https://github.com/sapegin/grunt-webfont/issues/219#issuecomment-73234070

The undocumented `normalize: true` option generated all my icons at the proper size.

**Comment by thybzi** [comment]
@hprofit for some reason it doesn't work for me, so I need to use some external tool for such conversion, such as `grunt-svg-modify` (if it could do the thing)

**Comment by yoksel** [comment]
Plugin resizes symbols via CSS. Do you need to change width and height attributes of particular SVG files?

**Comment by thybzi** [comment]
Yes, I need to rescale the paths. So, this plugin doesn't suit my needs?

**Comment by localBlackMage** [comment]
@thybzi 

This plugin will spit out font files for your SVGs which you can resize (easily) via font-size. The problem I was having was that it seemed to normalize all my SVGs to a really small (1x1 px) size so I'd have to blow them up in order to see them which was a whole other can of worms. 

Mucking around through https://github.com/sapegin/grunt-webfont/issues/219 I found the comment regarding an undocumented option that will normalize all of the SVGs to the same size and then (presumably) make them all 32x32 px. 

Does that describe the problem you're having?
